### PR TITLE
Add Applicant model and role field in User model

### PR DESCRIPTION
**Pull request by Palwisha-18**
- added user profile view

### CODE REVIEW

File: app/core/models.py [hunk 1]
```diff
@@ -15,9 +15,19 @@
 
 class User(AbstractBaseUser, PermissionsMixin):
     """User in the system."""
+
```
Comment (ahmadjanan) [inline app/core/models.py:28]: I think we should create a Nested class named `Role` here, with constants for values:
```
ADMIN = 'PortalAdmin'
RECRUITER = 'Recruiter'
APPLICANT = 'Applicant'
# with the following choices
CHOICES = (
        ('Admin', ADMIN),
        ('Recruiter', RECRUITER),
        ('Applicant', APPLICANT),
)
```

File: app/core/models.py [hunk 2]
```diff
@@ -49,3 +59,16 @@ class Recruiter(models.Model):
     company = models.ForeignKey(Company, default=None, null=True, blank=True, on_delete=models.SET_NULL,
                                 related_name='recruiters')
     is_admin = models.BooleanField(default=False)
+
+
+class Applicant(models.Model):
+    GENDER_CHOICES = (
```
Comment (ahmadjanan) [inline app/core/models.py]: Same class here as above comment.

**Review by ahmadjanan — CHANGES_REQUESTED** [review]

File: app/core/models.py (hunk 1 above)
Comment (Palwisha-18) [reply app/core/models.py:28]: updated 👍

File: app/core/models.py (hunk 2 above)
Comment (Palwisha-18) [reply app/core/models.py]: updated 👍

**Review by ahmadjanan — APPROVED** [review]
LGTM 👍

**Pull request merged**